resources/views/beatmap_discussions/index.blade.php

```diff
@@ -63,6 +83,16 @@ class="simple-form__input"
                     </div>
                 </div>
 
+                <div class="simple-form__row simple-form__row--no-label">
```

Comment:
the padding seems a bit off

![](https://s.myconan.net/2019-10/firefox_2019-10-29_18-41-53.png)